src/openssl_auth.cpp

```diff
@@ -26,7 +26,6 @@
 #include <cstdlib>
 #include <cerrno>
 #include <mutex>
-#include <pthread.h>
 #include <unistd.h>
```

Comment:
If you want to make the corrections you commented in subsequent reviews, you will need to `#include <thread>` here.